app/styles/main.scss

```diff
@@ -24,6 +24,15 @@ body {
   color: $color_mine_shaft_approx;
   margin: 0;
   font-size: $base_font_size;
+  a {
+    text-decoration: none;
+    cursor: pointer;
```

Comment:
this style was previously implemented (by you?) ... I did not intend to refactor existing styling decisions. However, I did refactor the code by separating 'a href' styling away from the class '.detail-toggle'